ti/instrumentation/utils.go

```diff
@@ -206,15 +206,21 @@ func getSplitTests(ctx context.Context, log *logrus.Logger, testsToSplit []ti.Ru
 	return testsToRun, nil
 }
 
-// getChangedFiles returns a list of files changed in the PR along with their corresponding status
-func getChangedFiles(ctx context.Context, workspace, lastSuccessfulCommitID string, isPushTrigger bool, log *logrus.Logger) ([]ti.File, error) {
-	diffFilesCmdFinal := diffFilesCmd
-	if isPushTrigger {
-		diffFilesCmdPushTrigger = append(diffFilesCmdPushTrigger, lastSuccessfulCommitID)
-		diffFilesCmdFinal = diffFilesCmdPushTrigger
-	}
+// getChangedFilesPR returns a list of files changed with their corresponding status for a PR.
+func getChangedFilesPR(ctx context.Context, workspace string, log *logrus.Logger) ([]ti.File, error) {
+	return getChangedFiles(ctx, workspace, log, diffFilesCmdPR)
+}
+
+// getChangedFilesPush returns a list of files changed with their corresponding status for push trigger/manual execution.
+func getChangedFilesPush(ctx context.Context, workspace, lastSuccessfulCommitID, currentCommitID string, log *logrus.Logger) ([]ti.File, error) {
+	diffFilesCmd := diffFilesCmdPush
```

Comment:
nit: remove this line and use push one in the line below directly

Reply:
golint and gocritic wont let me do that